### PR TITLE
Re-enabled Node 12 on Travis

### LINKED ISSUE #40: tests failing on Node 11 on travis
The tests are failing on Node 11 on travis.

I cannot understand why, so I'm disabling them.

It would be fantastic if someone could pick this up.

### PR DESCRIPTION
**Pull request by MichielDeMey**
Re-enabling Node 12 on Travis, attempt to fix #40

### CODE REVIEW

**Comment by mcollina** [comment]
https://travis-ci.org/fastify/fastify-postgres/jobs/492216929 < here is the failure

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 language: node_js
 
 node_js:
+  - "11"
```
Comment (darkgl0w) [inline .travis.yml]: Hello,
You can drop Node.js 11 for Node.js 12.
```suggestion
  - "12"
```

**Comment by MichielDeMey** [comment]
Tests are passing! 😎

**Comment by MichielDeMey** [comment]
Let’s merge this massive PR! 🏋️‍♂️😂

**Pull request merged**